jax/scipy/linalg.py

```diff
@@ -264,3 +264,101 @@ def tril(m, k=0):
 @_wraps(scipy.linalg.triu)
 def triu(m, k=0):
   return np.triu(m, k)
+
+@_wraps(scipy.linalg.expm, lax_description=textwrap.dedent("""\
+It does not take advantage of the sparsity of `A`. In `scipy.linalg.expm`, the sparsity is exploited.
```

Comment:
Does scipy.linalg.expm really make use of sparsity structure? That seems a little surprising to me — I would expect that only for scipy.sparse.linalg.expm.

Reply:
I see now that scipy.linalg.expm is implemented as an alias to scipy.sparse.linalg.expm.

But note that JAX doesn’t have sparse matrices (yet), so I’m not entirely sure this notice makes sense.

Reply:
Yes, I was trying to indicate that we are not aliasing a sparse version.  I will remove the notice.